Makefile

```diff
@@ -45,7 +45,12 @@ exes: $(EXES)
 
 install: lib_install #drv_install
 
-lib_install:
+makedirs:
+        @ echo "creating $(DESTLIB) ..."
+        mkdir -p $(DESTLIB)
```

Comment:
isn't /usr/bin/install supposed to create directories when necessary? on my RHEL6:
```
$ install --help
...
-D                  create all leading components of DEST except the last,
                        then copy SOURCE to DEST
```